### ISSUE TITLE
Bake into stylint core

### ISSUE THREAD

**Issue by SimenB** [sertae/stylint-json-reporter#3]
Having some way of getting easily parseable data out of stylint out of the box would be nice.

Would make instructions for sertae/stylint-plugin easier, and more in line with the other linters

**Comment by wojciechczerniak** [comment]
True. With SimenB/stylint#356 this reporter will be as easy as `JSON.stringify`. After all it will be a javascript object with correct structure. Eslint does this in their [json formatter](https://github.com/eslint/eslint/blob/master/lib/formatters/json.js). 

👍

**Comment by SimenB** [comment]
Yup, looks perfect
